reconcile/openshift_saas_deploy.py

```diff
@@ -120,6 +131,9 @@ def run(
     # - this is not a dry run
     # - there is a single saas file deployed
     notify = not dry_run and len(saas_files) == 1
+    skip_successful_deploy_notifications = False
+    if len(saas_files) > 0:
+        skip_successful_deploy_notifications = saas_files[0].skip_successful_deploy_notifications
```

Comment:
Can be written as

```python
skip_successful_deploy_notifications = saas_files[0].skip_successful_deploy_notifications if saas_files else False
```

Reply:
`saas_files` already check in above code and raised exception, and `notify` will only be set to `True` if there is only one saas_file, so  `skip_successful_deploy_notifications` is not needed here, can directly use `saas_file.skip_successful_deploy_notifications` in `slack_notify` calls.

Reply:
code has changed, to clarify what I mean:

* https://github.com/app-sre/qontract-reconcile/pull/3913/files#diff-304fd1216ea8201e3242ba749fcc1bebbe71e61fd70851f7ed266e7318ed40b2R136-R138 is not needed
* https://github.com/app-sre/qontract-reconcile/pull/3913/files#diff-304fd1216ea8201e3242ba749fcc1bebbe71e61fd70851f7ed266e7318ed40b2R170 and https://github.com/app-sre/qontract-reconcile/pull/3913/files#diff-304fd1216ea8201e3242ba749fcc1bebbe71e61fd70851f7ed266e7318ed40b2R184 can directly use `saas_file.skip_successful_deploy_notifications`